easybuild/easyconfigs/p/PyBEL/PyBEL-0.15.5-gfbf-2023a.eb

```diff
@@ -0,0 +1,74 @@
+easyblock = 'PythonBundle'
+
+name = 'PyBEL'
+version = '0.15.5'
+
+homepage = 'https://github.com/pybel/pybel'
+description = """
+PyBEL is a pure Python software package that parses BEL (Biological Expression Language)
+documents, validates their semantics, and facilitates data interchange between common formats
+and database systems like JSON, CSV, Excel, SQL, CX, and Neo4J.
+"""
+
+toolchain = {'name': 'gfbf', 'version': '2023a'}
+
+builddependencies = [
+    ('hatchling', '1.18.0'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('Python-bundle-PyPI', '2023.06'),
+    ('networkx', '3.1'),
+    ('SQLAlchemy', '2.0.25'),
+    ('tqdm', '4.66.1'),
+    ('pydantic', '2.5.3'),  # for bioregistry
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('multisplitby', '0.0.1', {
+        'checksums': ['e40489fa174fd7afd28a895477e6eefc64d6cfd863cca6900530e436a40e356f'],
+    }),
+    ('bel_resources', '0.0.3', {
+        'checksums': ['ec2d6390f89c0a98d67868bf02f7538709df0ba5b5ba6ddeb105512da07e82b5'],
+    }),
+    ('PyTrie', '0.4.0', {
+        'checksums': ['8f4488f402d3465993fb6b6efa09866849ed8cda7903b50647b7d0342b805379'],
+    }),
+    ('curies', '0.7.9', {
+        'checksums': ['3b63c5fea7b0e967629a3a384b1a8c59b56c503487c1dcbacddeab59e25db4d8'],
+    }),
+    ('more_click', '0.1.2', {
+        'checksums': ['085da66d5a9b823c5d912a888dca1fa0c8b3a14ed1b21ea9c8a1b814857a3981'],
+    }),
+    ('bioregistry', '0.11.10', {
+        'checksums': ['d3e155ba38d88f5e4a84a2d337c33109f3434f6b2e64a285eba14817ed5c4492'],
+    }),
+    ('click-plugins', '1.1.1', {
+        'checksums': ['46ab999744a9d831159c3411bb0c79346d94a444df9a3a3742e9ed63645f264b'],
+    }),
+    ('humanize', '4.8.0', {
+        'checksums': ['9783373bf1eec713a770ecaa7c2d7a7902c98398009dfa3d8a2df91eec9311e8'],
+    }),
+    ('psycopg2-binary', '2.9.9', {
+        'modulename': 'psycopg2',
+        'checksums': ['7f01846810177d829c7692f1f5ada8096762d9172af1b1a28d4ab5b77c923c1c'],
+    }),
+    ('pystow', '0.5.4', {
+        'checksums': ['2692180cb405bd77259bee6c7f4db545d10e81939980064730609f21750567ff'],
+    }),
+    ('ratelimit', '2.2.1', {
+        'checksums': ['af8a9b64b821529aca09ebaf6d8d279100d766f19e90b5059ac6a718ca6dee42'],
+    }),
+    ('requests_file', '2.1.0', {
+        'checksums': ['0f549a3f3b0699415ac04d167e9cb39bccfb730cb832b4d20be3d9867356e658'],
+    }),
+    ('pybel', version, {
+        'checksums': ['c2aabd0ad65b9f205f9ed6eae126d19875c1312f8e35641f4f42bcc1fc573f9e'],
+    }),
+]
+
```

Comment:
this installs some executables, add e.g. `sanity_check_commands = ["pybel --help"]`?